internal/cmd/operator-sdk/run/cmd.go

```diff
@@ -34,7 +35,7 @@ Currently only the package manifests format is supported via the 'packagemanifes
 
 	cmd.AddCommand(
 		// TODO(joelanford): enable bundle command when implementation is complete
-		// bundle.NewCmd(cfg),
+		bundle.NewCmd(cfg),
```

Comment:
We are adding the command actually and its tests. In this way, it requires a fragment.

Reply:
Maybe this would be removed from this PR. I think this command will be uncommented and the related changes in the doc would be made along with the implementation PR. I'm not sure if that would be done in this PR. I had uncommented this temporarily here to skip the errors. Do I include a changelog fragment for the e2e test though ?

Reply:
For the e2e we should not add changelog fragment. 
However, I understand that the task here is to make available the command `run bundle` and add the tests. we will unable to merge the tests without the command. So, the fragment would be to make clear that we are `adding a new command` on the tool. Is it make sense?